html/index.html

```diff
@@ -0,0 +1,261 @@
+<html>
+<head>
+<title>CarCompany</title>
+<meta http-equiv="Content-Type" content="text/html; charset="utf-8">
+<link href="style.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+
+<table id="t1">
+	<tr>
+		<td colspan=5 rowspan=3>
+			<img id="l1" src="images/layout_01.gif" alt=""> </td>
+		<td rowspan=3>
+			<img id="l2" src="images/layout_02.gif" alt=""></td>
+		<td colspan=8>
+			<img id="l3" src="images/layout_03.gif" alt=""></td>
+		<td width="100%" rowspan=7 background="images/layout_04.gif">
+			<img id="l4" src="images/layout_04.gif" alt=""></td>
+	</tr>
+	<tr>
+		<td colspan=8 background="images/layout_05.gif" height="18">
+		
+                <b> 
+            <a id="a1" href="index.htm"> HOME </a>  *
```

Comment:
### Please be more patient

It looks like your file name is _index.html_ NOT _index.htm_